core/src/main/java/org/eclipse/hono/util/ResourceIdentifier.java

```diff
@@ -154,6 +179,20 @@ public static ResourceIdentifier from(final String endpoint, final String tenant
         return new ResourceIdentifier(endpoint, tenantId, resourceId);
     }
 
+    /**
+     * Creates a resource identifier for an endpoint from an other resource identifier, a tenantId and a resourceId.
```

Comment:
the purpose of this is unclear to me. The description should provide a hint on why you would want to do this ...

Reply:
Ok